### PR TITLE
Upgrade hasha, drop node.js 6

### PR DESCRIPTION
**Pull request by coreyfarrell**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -45,11 +34,9 @@ const tryReadFile = {
 
 const tryExecFile = {
   async (file, args, options) {
-    return new Promise(resolve => {
-      cp.execFile(file, args, options, (err, stdout) => {
-        resolve(err ? null : stdout)
-      })
-    })
+    return promisify(cp.execFile)(file, args, options)
```
Comment (novemberborn) [inline index.js]: Is there any point in storing the promisified function?

File: .travis.yml [hunk 2]
```diff
@@ -7,17 +7,13 @@ node_js:
   - 10
   - 8
 env:
-  - FRESH_DEPS=false
   - FRESH_DEPS=true
 matrix:
-  exclude:
+  include:
+    # Only verify package-lock.json on node.js 11 for linux
```
Comment (novemberborn) [inline .travis.yml]: This isn't about verifying the package lock, or at least not how I would understand it. The idea is that the package lock gives us a predictable CI run, but we do at least one run with the newest dependencies. Probably more useful for a bigger project like AVA than this module though.

File: index.js (hunk 1 above)
Comment (coreyfarrell) [reply index.js]: It would be a tiny optimization.

File: .travis.yml (hunk 2 above)
Comment (coreyfarrell) [reply .travis.yml]: The problem is that it does verify the package-lock.json and the file is different between node.js 11 and all previous versions.

Comment (novemberborn) [reply .travis.yml]: Ah yes of course. Let's just delete the package lock and remove the CI checks. Best disable it too (like https://github.com/sindresorhus/p-defer/blob/master/.npmrc).

**Comment by novemberborn** [comment]
Feel free to replace the linter too, by the way. `standard` will probably cause the least amount of churn. Or not, if it works it works 😄

**Comment by coreyfarrell** [comment]
If I were to make the effort to change linters it would be `xo`, my C background makes me want semicolons and xo is closest to the style I already use in C projects.  That would be a large change, I'm fine just leaving it as is.

File: .travis.yml (hunk 2 above)
Comment (coreyfarrell) [reply .travis.yml]: Done

**Review by novemberborn — APPROVED** [review]
Fair enough. :shipit:!

**Pull request merged**